Header      header
geometry_msgs/Pose pose
RssiStamped rssi
